projects/inkscape/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -eu
+nproc=$(nproc)
+
+### backup flags ###
+
+OLDCFLAGS=${CFLAGS}
+OLDCXXFLAGS=${CXXFLAGS}
+OLDSANITIZER=${SANITIZER}
+OLDCOVERAGE_FLAGS=${COVERAGE_FLAGS}
+CFLAGS="-lpthread"
```

Comment:
Why do you disable the sanitizers when building deps? 
For asan/ubsan it's fine -- just asking. 
If you want msan you'll need  to use the sanitizer flags.

Reply:
As far as I understood what happened, some deps built tools that they ran during the build, and the presence of these flags made them fail

Reply:
I saw leaks while building popular packages. For this purpose some of the `build.sh` files in oss-fuzz disable lsan (look for `ASAN_OPTIONS=detect_leaks=0`)
Maybe this would be enough. But -- this is unlikely the cause of your current problem